test/functional/ui/statuscolumn_spec.lua

```diff
@@ -111,6 +117,9 @@ describe('statuscolumn', function()
       16 12│aaaaa                                          |
                                                            |
     ]])
+    command([[set stc=%l%=%{&rnu?'\ ':''}%r│]])
+    screen:expect_unchanged()
+    command([[set stc=%{&nu?v:lnum:''}%=%{&rnu?'\ '.v:relnum:''}│]])
```

Comment:
Thank you for looking for relevant existing tests to update. It's often counterproductive to add entirely new tests that differ very little from existing tests, especially if the tests are verbose.